style.css

```
@@ -228,3 +229,72 @@ body {
     width: 20px;
     margin-right: 10px;
 }
+
+.information-footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    color: #42446E;
+    font-family: DM Sans;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+}
+
+.information-footer .contacts-foolter {
+    display: flex;
+    align-items: center;
+    margin-bottom: 62px;
+    justify-content: space-between;
+}
+
+.information-footer .contacts-foolter .content-foolter {
+    display: flex;
+    gap: 20px;
+}
+
+.information-footer .contacts-foolter .content-foolter .text-footer {
+    margin-right: 63px;
+}
+
+
+.navigation-footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 56px; /* Pq não aplicou o margin-bottom?*/
+    margin-bottom: 60px;
+}
+
+.navigation-footer .navigation-end .link-footer {
+    font-family: DM Sans;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+    text-decoration: none;
+    margin-right: 56px;
+}
+
+.navigation-footer .division {
+    stroke-width: 2px;
+    stroke: var(--dark-content, #666);
```

Não precisamos usar `var` aqui.
```suggestion
    stroke: #666;
```

---

@mikaelelias

---

feito!